src/components/CreateEventDialog.tsx

```
@@ -1307,6 +1308,54 @@ export function CreateEventDialog({
+                  type="button"
+                  variant="outline"
+                  onClick={() => {
+                    const current = form.getValues("ratingMetrics") || [];
+                    form.setValue("ratingMetrics", [...current, ""]);
+                  }}
+                  className="w-full border-dashed font-mono text-xs font-bold"
+                >
+                  <Plus className="mr-1 h-3 w-3" /> Add Rating Dimension
+                </Button>
+                <p className="text-xs text-black/50">
+                  After the event, attendees rate these categories on a 0-100 slider. Leave empty
```

ESLint

[error] Insert `·to`

(prettier/prettier)